docs/users/step-debugging.md

```diff
@@ -1,90 +1,90 @@
-  ![Start listening for debug connections button](images/phpstorm_listen_for_debug_connections.png)
-2. Set a breakpoint.
-3. Visit a page that should stop in the breakpoint you set.
-4. PhpStorm will ask you what mapping to use, so it can figure out how to map the path inside the container to the path on your workstation. The first time you do this with a project, PhpStorm will pop up a "server" dialog box to set the mapping. The default will work, but it's best to click the checkbox to map the whole project directory to /var/www/html.
+### Debug a website
 
-Note when using this recommended option:
+To start and stop Xdebug from the browser, you need to set a cookie parameter. You can use one of the
+[available browser debugging extensions](#Available browser debugging extensions). Alternatively you can debug every
+request by executing `ddev xdebug autostart`.
 
-1. Please use the latest DDEV version.
-2. Under Run >> Edit Configurations, check that there are no "Servers" already defined. PhpStorm will create a new "Server" (file mapping) for you as discussed above, but only if you don't already have one. You can delete all servers and have PhpStorm create a new one, or you can create/edit an existing server as discussed below.
+Activate the debugging extension in your browser:
 
-#### PhpStorm "Run/Debug configuration" Debugging
+![Xdebug helper in Chrome](images/ps_xdebug_helper_chrome.png)
 
-PhpStorm [run/debug configurations](https://www.jetbrains.com/help/phpstorm/creating-and-editing-run-debug-configurations.html) require slightly more up-front work but can offer more flexibility and may be easier for some people.
+#### Available browser debugging extensions
 
-1. Under the "Run" menu select "Edit configurations"
-2. Click the "+" in the upper left and choose "PHP Web Application" to create a configuration. Give it a reasonable name.
-3. Create a "server" for the project. Make sure that "Name" is exactly the same as your "host" (e.g. `my-site.ddev.site`) (Screenshot below)
-4. Add file mappings for the files on the server. Click on the local repo path and add "/var/www/html" as the "Absolute path on the server" and your repository root as the path on the host.
-5. Set an appropriate breakpoint.
-6. Start debugging by clicking the "debug" button, which will launch a page in your browser.
+| Chrome                                                                                        | Firefox                                                                                                                                                                        | Internet Explorer                                                               | Microsoft Edge                                                                                                    | Safari                                                                             | Opera                                                                           |
```

Comment:
At this point IE is just fluff I think. You're the Windows expert so I defer to you, but consider removing.

Reply:
Support was dropped on june 15th 2022. Just a few days ago - finally. I agree, will remove it.